### ISSUE TITLE
Improve managed and common rule aliases Error messaging

### ISSUE THREAD

**Issue by aidanmelen** [aidanmelen/terraform-aws-security-group-v2#29]
**Is your feature request related to a problem? Please describe.**

The following example:

```hcl
#tfsec:ignore:aws-ec2-no-public-egress-sgr
module "security_group" {
  source  = "aidanmelen/security-group-v2/aws"
  version = "2.0.0"

  name        = local.name
  description = "Allow TLS inbound traffic"
  vpc_id      = data.aws_vpc.default.id

  # recommended
  unpack = true

  ingress = [
    {
      rule             = "this-is-not-a-rule"
      cidr_blocks      = [data.aws_vpc.default.cidr_block]
      ipv6_cidr_blocks = [data.aws_vpc.default.ipv6_cidr_block]
    }
  ]

  egress = [
    { rule = "all-all-to-public" }
  ]
}
```

with produce these Terraform Errors:

```bash
╷
│ Error: Missing required argument
│ 
│   with module.security_group.aws_security_group_rule.ingress["172.31.0.0/16"],
│   on ../../main.tf line 89, in resource "aws_security_group_rule" "ingress":
│   89:   from_port                = try(each.value.from_port, null)
│ 
│ The argument "from_port" is required, but no definition was found.
╵
╷
│ Error: Missing required argument
│ 
│   with module.security_group.aws_security_group_rule.ingress["2600:1f13:b21:6500::/56"],
│   on ../../main.tf line 89, in resource "aws_security_group_rule" "ingress":
│   89:   from_port                = try(each.value.from_port, null)
│ 
│ The argument "from_port" is required, but no definition was found.
╵
╷
│ Error: Missing required argument
│ 
│   with module.security_group.aws_security_group_rule.ingress["172.31.0.0/16"],
│   on ../../main.tf line 90, in resource "aws_security_group_rule" "ingress":
│   90:   to_port                  = try(each.value.to_port, null)
│ 
│ The argument "to_port" is required, but no definition was found.
╵
╷
│ Error: Missing required argument
│ 
│   with module.security_group.aws_security_group_rule.ingress["2600:1f13:b21:6500::/56"],
│   on ../../main.tf line 90, in resource "aws_security_group_rule" "ingress":
│   90:   to_port                  = try(each.value.to_port, null)
│ 
│ The argument "to_port" is required, but no definition was found.
╵
╷
│ Error: Missing required argument
│ 
│   with module.security_group.aws_security_group_rule.ingress["172.31.0.0/16"],
│   on ../../main.tf line 91, in resource "aws_security_group_rule" "ingress":
│   91:   protocol                 = try(each.value.protocol, null)
│ 
│ The argument "protocol" is required, but no definition was found.
╵
╷
│ Error: Missing required argument
│ 
│   with module.security_group.aws_security_group_rule.ingress["2600:1f13:b21:6500::/56"],
│   on ../../main.tf line 91, in resource "aws_security_group_rule" "ingress":
│   91:   protocol                 = try(each.value.protocol, null)
│ 
│ The argument "protocol" is required, but no definition was found.
╵
```

**Describe the solution you'd like**

It would be great to tell the user the specific rule alias(s) lookup(s) that failed.

**Comment by aidanmelen** [comment]
Fix coming in `v2.0.1`. Improved output:

```bash
╷
│ Error: Incorrect attribute value type
│ 
│   on ../../main.tf line 89, in resource "aws_security_group_rule" "ingress":
│   89:   from_port                = try(each.value.from_port, null)
│     ├────────────────
│     │ each.value.from_port is "The rule alias is invalid: this-is-not-a-rule. https://github.com/aidanmelen/terraform-aws-security-group-v2#rule-aliases"
│ 
│ Inappropriate value for attribute "from_port": a number is required.
╵
╷
│ Error: Incorrect attribute value type
│ 
│   on ../../main.tf line 89, in resource "aws_security_group_rule" "ingress":
│   89:   from_port                = try(each.value.from_port, null)
│     ├────────────────
│     │ each.value.from_port is "The rule alias is invalid: this-is-not-a-rule. https://github.com/aidanmelen/terraform-aws-security-group-v2#rule-aliases"
│ 
│ Inappropriate value for attribute "from_port": a number is required.
```

**Comment by aidanmelen** [comment]
https://github.com/aidanmelen/terraform-aws-security-group-v2/releases/tag/v2.0.1
